components/result-count-title/src/index.js

```diff
@@ -4,16 +4,19 @@ import styled from 'react-emotion';
 
 import ResultCount from '@govuk-frederic/result-count';
 
-const Wrapper = styled('div')({
-  display: 'inline-block',
-});
+const Wrapper = styled('div')(
+  {
+    display: 'flex',
```

Comment:
this change may have a negative knock-on effect in some other places where `ResultCount` is used

Reply:
Right, this can actually go back, I see now - and it fixes the padding issue noted below. Reverted.